main.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+from patblib import Path
+
+
+def z(x, y):
+    x = np.array(x)
+    return (y - np.mean(x)) / np.std(x)
```

Comment:
Consider setting `(y - np.mean(x)) / np.std(x)` as a variable (a descriptive one) and then returning that variable. This can help with readability of the code.

Reply:
I set this equal to z and then returned z here 45cd48dd64bae55521182d52a6cf225c0d39076b